components/server/src/server.rs

```diff
@@ -1874,9 +1877,31 @@ impl EnginesResourceInfo {
         if let Some(raft_engine) = &self.raft_engine {
             fetch_engine_cf(raft_engine, CF_DEFAULT, &mut normalized_pending_bytes);
         }
-        for cf in &[CF_DEFAULT, CF_WRITE, CF_LOCK] {
-            fetch_engine_cf(&self.kv_engine, cf, &mut normalized_pending_bytes);
+
+        // Record the compaction pending bytes of the tablet with the latest suffix for
+        // each region
+        let mut pending_bytes_record = HashMap::<_, (_, _)>::new();
```

Comment:
Put the hashmap inside `EnginesResourceInfo` so that we don't need to reallocate every few seconds.

Reply:
If one tablet is destroyed, how can we remove it from the hashmap?

Reply:
clear it before doing the scan.